### PR TITLE
fix: add kratos redirect endpoint middleware

### PR DESCRIPTION
**Pull request by rebelchris**
Changing the redirect URI for github to the SSO domain

### CODE REVIEW

**Review by idoshamun — COMMENTED** [review]
Have you tested it locally?

File: src/config.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ const config = {
     clientSecret: process.env.GITHUB_CLIENT_SECRET,
     authorizeUrl: 'https://github.com/login/oauth/authorize',
     authenticateUrl: 'https://github.com/login/oauth/access_token',
+    redirect_uri: 'https://sso.daily.dev/api/github_redirect',
```
Comment (idoshamun) [inline src/config.js:104]: It's better to extract it from an environment variable to support multiple environments

Comment (rebelchris) [reply src/config.js:104]: Sure thing, will do.

Not really tested it end to end as we can't set it.
But let me try and see if it atleast goes to kratos

Comment (idoshamun) [reply src/config.js:104]: We must test it locally before we push it. We need to the test legacy system

**Pull request merged**